linalg/hypre.cpp

```diff
@@ -3144,8 +3197,13 @@ void HypreSmoother::SetOperator(const Operator &op)
       }
       else
       {
+// TODO: replace hypre_ParCSRMatrixOwnsRowStarts with MFEM_HYPRE_VERSION > 22200
+#ifdef hypre_ParCSRMatrixOwnsRowStarts
```

Comment:
This should be a temporary fix until we don't have a new release (greater than 2.22.0) in hypre.